project/Doc.scala

```diff
@@ -50,7 +50,7 @@ object UnidocRoot extends AutoPlugin {
     inTask(unidoc)(Seq(
       unidocProjectFilter in ScalaUnidoc := projectsAndDependencies(scaladslProjects),
       unidocProjectFilter in JavaUnidoc := projectsAndDependencies(javadslProjects),
-      apiMappings in ScalaUnidoc := (apiMappings in (Compile, doc)).value
+      autoAPIMappings in ScalaUnidoc := true
```

Comment:
Using `apiMappings in ScalaUnidoc := (apiMappings in (Compile, doc)).value`  instead of `autoAPIMappings in ScalaUnidoc := true` wasn't creating the x-references.
This seems to work if the upstream dep has the `apiURL` setting properly set.

Reply:
Akka and Akka HTTP already set `apiURL` so x-referencing works as expected with those classes. Except I noticed something weird with `Sink`: (not linked?)

![screen shot 2018-03-27 at 11 33 09](https://user-images.githubusercontent.com/762126/37959411-53bcf404-31b3-11e8-8587-2846220048ac.png)

Note how `Source` and `NotUsed` are linked and `Sink` isn't. I tested the links for `Source` and `NotUsed` and they work as expected. I also tested a link to `ActorSystem`. I am not sure `Sink` is the only issue, though.

Reply:
We started setting `apiURL` some releases ago. Glad it is being used here. :)

Reply:
From the screenshot above I see that the `Futrue` type is not cross linked. The https://github.com/ThoughtWorksInc/sbt-api-mappings plugin might be useful here.

Reply:
Hmm, `Boolean` is linked though.

I'll have a look at `Future` but maybe on a separate PR. Thanks for the heads up.

Reply:
While reviewing the issue with `Future` and `Sink` I've observed the following two cases for Scala API classes:

* Link exists and works as expected: `Option`, `ExecutionContext`
* Link is not added but mouse hovering displays an `alt` text with appropriate FQCN: `List`, `Future`

(Attached screenshot for all the cases described above)
![screen shot 2018-03-28 at 19 49 54](https://user-images.githubusercontent.com/762126/38047005-686200de-32c1-11e8-8f87-94b1904b69a7.png)

This is similar to what I observed with Akka where `Sink` is the only Akka class that's not linked but in the case of Scala there are a few more unlinked types.